travis-ci/cleanup-integration-tests.sh

```diff
@@ -19,15 +19,23 @@ set +e
     ./node_modules/.bin/kes cf delete \
       --kes-folder app \
```

Comment:
Is this still necessary?

Reply:
Hmm, good question.

Reply:
Yes - it still wants a config file to load, even if it doesn't use it: 

```
 ->./node_modules/.bin/kes cf delete --region us-east-1 --stack jk5-iam --yes
No Kes override found at /Users/kovarik/workspace/cumulus/example/.kes/kes.js, continuing
No Kes override found at /Users/kovarik/workspace/cumulus/example/path/to/nowhere/kes.js, continuing
ENOENT: no such file or directory, open '/Users/kovarik/workspace/cumulus/example/.kes/config.yml'
```